src/apps/answers/crud/answers.py

```diff
@@ -234,6 +234,8 @@ async def get_respondents_submit_dates(
         query = query.where(func.date(AnswerSchema.created_at) >= filters.from_date)
         query = query.where(func.date(AnswerSchema.created_at) <= filters.to_date)
         query = query.where(AnswerSchema.applet_id == applet_id)
+        query = query.where(AnswerSchema.activity_history_id.contains(filters.activity_id))
+
```

Comment:
Please, use `AnswerSchema.id_from_history_id(AnswerSchema.activity_history_id) == str(filters.activity_id)`

this AnswerSchema.id_from_history_id() "translate" history id to activity id